common/src/main/resources/assets/wynntils/lang/en_us.json

```diff
@@ -717,6 +719,8 @@
   "feature.wynntils.itemHighlight.zeroStarIngredientHighlightColor.name": "0 Star Ingredient Highlight Color",
   "feature.wynntils.itemHighlight.zeroStarIngredientHighlightEnabled.description": "Enable highlighting of 0 star ingredient items",
   "feature.wynntils.itemHighlight.zeroStarIngredientHighlightEnabled.name": "0 Star Ingredient Highlight",
+  "feature.wynntils.itemInteractionSound.description": "Plays sounds when items are moved around in inventories.",
```

Comment:
Is this a planned further addition to this PR, or something that never happened/got removed?